### DISCUSSION TITLE
InfluxDB und Performancebedenken?

### DISCUSSION THREAD

**Discussion by gereons** [solectrus/solectrus#3393] [General]
Georg schrieb u.a. [hier](https://github.com/orgs/solectrus/discussions/3392#discussioncomment-10463982) aber ähnlich auch schon anderswo:

> Das könnte man zwar hochdrehen, was aber den Traffic auf der InfluxDB sehr erhöhen würde, was ich gerne vermeiden möchte.

Und ich frage mich: ist das wirklich so ein wichtiger Punkt, speziell für diejenigen die Solectrus auf einem "richtigen" Server bei Hetzner oder sonstwo betreiben? Ich tu das, und die Kiste langweilt sich zu Tode:

```
solectrus:~# uptime
16:52:47 up 15 days,  3:54,  1 user,  load average: 0.15, 0.26, 0.12
```

Die CPU-Zeile in `top` sieht auch praktisch immer etwa so aus: 
```
%Cpu(s):  2.9 us,  5.9 sy,  0.0 ni, 91.2 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
``` 
also jedenfalls immer >90% idle und sehr kleine Werte beim I/O-wait und das heisst doch eindeutig, dass die Maschine noch ordentlich Kapazität für andere Sachen hat und man _erheblich_ mehr machen könnte ohne dass es Probleme gibt. 

Oder?

**Comment by ledermann** [comment]
Meine Aussage basiert nicht auf durchgeführten Lasttests, sondern ist eher als reine Vorsichtsmaßnahme zu sehen.

Grundsätzlich hast du natürlich recht - zumindest ein zeitgemäßer Server wird durch SOLECTRUS (und insbesondere InfluxDB) kaum ausgelastet. Aber ich möchte auch weiterhin sicherstellen, dass ein kleiner Raspi nicht überfordert wird.

Für den konkreten Fall des Power-Splitters habe ich [vorhin](https://github.com/orgs/solectrus/discussions/3330#discussioncomment-10473040) die Möglichkeit ergänzt, dass man die Berechnung häufiger durchführen kann.

**Reply by gereons** [reply]
Sehr schön, danke!
